src/themes/teams-dark/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,9 @@
+export interface TreeTitleVariables {
```

Comment:
please, don't duplicate interface definitions - consider to use the one defined for a default theme, similar to how it is done for `buttonVariables`: https://github.com/stardust-ui/react/blob/master/src/themes/teams-dark/components/Button/buttonVariables.ts#L4